s/elements/stickpad/style.css.ts

```diff
@@ -0,0 +1,25 @@
+
+import {css} from "@chasemoskal/magical"
+
+export default css`
+	:host {
+		display: flex;
+		width: 30em;
+		height: 15em;
+		border: 1px solid;
+		touch-action: none;
```

Comment:
it seems suspicious that the stickpad would have `touch-action: none;`

is it actually necessary?

Reply:
if i dont add this then when trying to touch the stick on phone it will trigger scrolling the page or other touch events and it would simply not work, preventDefault doesnt seem to work

Reply:
> is it actually necessary?

yes, not having it causes other default browser events to interfere, which then triggers a `pointercancel` event

Reply:
okay cool, that makes sense :+1: